app/admin/quests/dashboard/[questId]/page.tsx

```
@@ -483,100 +483,117 @@ const handleTabChange = (pageOrUpdater: SetStateAction<number>) => {
-        } else if (step.type === "CustomApi") {
-          await AdminService.createCustomApi({
-            quest_id: questId.current,
-            name: step.data.api_name,
-            desc: step.data.api_desc,
-            api_url: step.data.api_url,
-            regex: step.data.api_regex,
-            href: step.data.api_href,
-            cta: step.data.api_cta,
-          });
+              })
+            );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)